nmostesting/suites/BCP0060102Test.py

```diff
@@ -778,11 +781,11 @@ def test_02(self, test):
                 question, possible_answers, test_type='multi_choice')['answer_response']
 
             if len(actual_answers) != len(expected_answers):
-                return test.FAIL('Incorrect Receiver identified')
+                return test.FAIL('Capable Receiver not identified')
             else:
                 for answer in actual_answers:
                     if answer not in expected_answers:
-                        return test.FAIL('Incorrect Receiver identified')
+                        return test.FAIL('Capable Receiver not identified')
```

Comment:
Hmm, actually, this is the `answer not in expected_answers` case, so that _is_ "Incorrect Receiver identified", isn't it?!

Reply:
Or was this meant to be pluralized, like the message in line 731? I guess that works sorta... (at least both cases are equivalent, right?)

Reply:
Yes, it should have been pluralized so will correct.  The intention was to reflect the intent of the test. i.e. "Identify capable Receivers" -> Failure -> "Capable Receivers not identified".

However, your point about the mechanics of the test not reflecting that intention is a fair criticism.  From a code point of view `answer not in expected_answers` does *not* describe the actual error condition.  The test mechanics have been "borrowed" from the IS04/05 Controller tests and I'm not sure the "question\answer" metaphor quite works here.

Perhaps renaming those parameters will aid clarity:
- `expected_answers` -> `capable_receivers`
- `actual_answers` -> `identified_receivers`
- `answer` -> `receiver`

We're then left with:
"Identify capable Receivers" -> `receiver not in capable_receivers` -> "Capable Receivers not identified".

Clearer do you think?

Reply:
No... `receiver not in capable_receivers` is not, as far as I can see, represented in English by "Capable Receivers not identified", but rather by "identified Receiver is incorrect because it's not capable".

Reply:
"Capable Receivers not correctly identified"?